src/pages/RecipeCalculator.vue

```diff
@@ -39,20 +39,48 @@
       <div id="version-group" class="p-2">
         <h3>Last Updated</h3>
         <p>v1.22</p>
-        <p>June 28, 2021</p>
+        <p>July 7, 2021</p>
       </div>
     </div>
   </div>
   <!-- Display Materials -->
   <div v-if="recipe && quantity" class="text-light">
     <h4>Materials</h4>
-    <recipe-tree
-      :label="materials.name"
-      :nodes="materials.materials"
-      :quantity="1"
-      :depth="0"
-      :toCraft="Number(quantity)"
-    />
+    <!-- Select Display Type -->
+    <select v-model="display" class="" id="recipe-selector">
+      <option value="">Select a View Type</option>
+      <option value="list">List View</option>
+      <option value="tree">Tree View</option>
+    </select>
+    <div v-if="display">
+      <div v-if="display === 'list'">
+        <div
+          class="tree-menu border-top border-bottom"
```

Comment:
Let's change this to `border-b` to avoid double thickness borders.

Reply:
`border-top border-bottom` changed to `border-b`, shows no border now

Reply:
Yeah this might be an inconsistency with bootstrap? Or rather I think you also need `border` and `border-b`? NBD

Reply:
Added `border-bottom` to the List View, couldn't get it to play nice with Tree View. But that should be fine if we'll be changing soon*